src/Compilers/CSharp/Portable/Symbols/Source/SourceDelegateMethodSymbol.cs

```diff
@@ -232,6 +232,7 @@ internal override LexicalSortKey GetLexicalSortKey()
         private sealed class InvokeMethod : SourceDelegateMethodSymbol
         {
             private readonly RefKind refKind;
+            private readonly DelegateDeclarationSyntax syntax;
```

Comment:
I'll opt into using the method location instead of the return type. Doesn't look worth it to me to (possibly) reparse the tree just to move the error location one token back.
The location of the delegate is descriptive.

---
In reply to: [115866901](https://github.com/dotnet/roslyn/pull/18715#discussion_r115866901) [](ancestors = 115866901)